db/mobile.php

```diff
@@ -35,9 +35,9 @@
                 'delegate' => 'CoreCourseModuleDelegate',
                 'method' => 'mobile_view_activity',
                 'styles' => [
-                    'url' => $CFG->wwwroot . '/mod/questionnaire/styles_app.css',
-                    'version' => '1.4'
-                ]
+                        'url' => $CFG->wwwroot . '/mod/questionnaire/mobile_css.php?v=2022092200',
```

Comment:
What is this change for? I'm not familiar with this.

Reply:
We will revert this change and just adjust the version. Please see the last comment for more detail.